pkg/cmdutil/cmdutil.go

```diff
@@ -38,3 +44,54 @@ func StringSliceToListStringWithQuotes(validOptions []string) string {
 	}
 	return listF
 }
+
+// IsURL accepts a string and determines if it is a URL
+func IsURL(s string) bool {
+	return strings.HasPrefix(s, "http:/") || strings.HasPrefix(s, "https:/")
+}
+
+// GetContentFromFileURL loads file content from the provided URL
+func GetContentFromFileURL(url string, ctx context.Context) (*os.File, error) {
+
+	req, err := http.NewRequestWithContext(ctx, "GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	client := http.DefaultClient
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("error loading file: %s", http.StatusText(resp.StatusCode))
+	}
+
+	respbody := resp.Body
+
+	defer resp.Body.Close()
+
+	tmpfile, err := ioutil.TempFile("", "rhoas_file-*")
+	if err != nil {
+		return nil, fmt.Errorf("error initializing temporary file: %w", err)
+	}
+
+	defer func() {
+		_ = tmpfile.Close()
+		_ = os.Remove(tmpfile.Name())
+	}()
+
+	_, err = io.Copy(tmpfile, respbody)
+	if err != nil {
+		return nil, err
+	}
+
+	specifiedFile, err := os.Open(tmpfile.Name())
```

Comment:
Logic here works but it is opening the same file twice.
It is better to 1) write content into that file. Set buffer to beginning 
```
 _, err = (*specifiedFile).Write(data)
	if err != nil {
		return nil, err
	}
	_, err = (*specifiedFile).Seek(0, io.SeekStart)
	if err != nil {
		return nil, err
	}
```